src/sentry/models/commitauthor.py

```diff
@@ -8,7 +8,7 @@
 
 class CommitAuthorManager(BaseManager):
     def get_or_create(self, organization_id, email, defaults, **kwargs):
-        # Force email address to lowercase because GitHub does this. Note though that this isn't technically
+        # Force email address to lowercase because many providers do this. Note though that this isn't technically
```

Comment:
I realize today this comment was incorrect so I fixed it (Github is the only provider which does not do this)